### ISSUE TITLE
Latest 15 Papers - May 10, 2025

### ISSUE THREAD

**Issue by KMnO4-zx** [KMnO4-zx/DailyArXiv#22]
**Please check the [Github](https://github.com/KMnO4-zx/DailyArXiv) page for a better reading experience and more papers.**

## Geoscience
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[GeoRSMLLM: A Multimodal Large Language Model for Vision-Language Tasks in Geoscience and Remote Sensing](http://arxiv.org/abs/2503.12490v1)** | 2025-03-16 | [papers-cool](https://papers.cool/arxiv/2503.12490v1) |  |
| **[Towards Foundation Models: Evaluation of Geoscience Artificial Intelligence with Uncertainty](http://arxiv.org/abs/2501.14809v1)** | 2025-01-15 | [papers-cool](https://papers.cool/arxiv/2501.14809v1) |  |
| **[When Geoscience Meets Foundation Models: Towards General Geoscience Artificial Intelligence System](http://arxiv.org/abs/2309.06799v5)** | 2024-11-12 | [papers-cool](https://papers.cool/arxiv/2309.06799v5) | <details><summary>accpe...</summary><p>accpeted by IEEE Geoscience and Remote Sensing Magazine</p></details> |
| **[Prototype-Based Methods in Explainable AI and Emerging Opportunities in the Geosciences](http://arxiv.org/abs/2410.19856v1)** | 2024-10-22 | [papers-cool](https://papers.cool/arxiv/2410.19856v1) | <details><summary>Accep...</summary><p>Accepted at AI for Science Workshop-Oral (Attention Track), Proceedings of 41st International Conference on Machine Learning (ICML) 2024</p></details> |
| **[Interpretable Geoscience Artificial Intelligence (XGeoS-AI): Application to Demystify Image Recognition](http://arxiv.org/abs/2311.04940v2)** | 2024-05-07 | [papers-cool](https://papers.cool/arxiv/2311.04940v2) | <details><summary>there...</summary><p>there are some erros in the results, and a newer revision is still preparing</p></details> |
| **[GeoGalactica: A Scientific Large Language Model in Geoscience](http://arxiv.org/abs/2401.00434v2)** | 2024-04-13 | [papers-cool](https://papers.cool/arxiv/2401.00434v2) |  |
| **[A novel approach for estimating functions in the multivariate setting based on an adaptive knot selection for B-splines with an application to a chemical system used in geoscience](http://arxiv.org/abs/2306.00686v3)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2306.00686v3) | 29 pages, 27 figures |
| **[When Geoscience Meets Generative AI and Large Language Models: Foundations, Trends, and Future Challenges](http://arxiv.org/abs/2402.03349v1)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2402.03349v1) |  |
| **[Differentiable modeling to unify machine learning and physical models and advance Geosciences](http://arxiv.org/abs/2301.04027v2)** | 2023-12-27 | [papers-cool](https://papers.cool/arxiv/2301.04027v2) |  |
| **[K2: A Foundation Language Model for Geoscience Knowledge Understanding and Utilization](http://arxiv.org/abs/2306.05064v2)** | 2023-09-13 | [papers-cool](https://papers.cool/arxiv/2306.05064v2) |  |
| **[AI Security for Geoscience and Remote Sensing: Challenges and Future Trends](http://arxiv.org/abs/2212.09360v2)** | 2023-06-22 | [papers-cool](https://papers.cool/arxiv/2212.09360v2) |  |
| **[Coarse-to-fine Task-driven Inpainting for Geoscience Images](http://arxiv.org/abs/2211.11059v3)** | 2022-12-06 | [papers-cool](https://papers.cool/arxiv/2211.11059v3) |  |
| **[Investigating the fidelity of explainable artificial intelligence methods for applications of convolutional neural networks in geoscience](http://arxiv.org/abs/2202.03407v2)** | 2022-09-05 | [papers-cool](https://papers.cool/arxiv/2202.03407v2) |  |
| **[Data Centred Intelligent Geosciences: Research Agenda and Opportunities, Position Paper](http://arxiv.org/abs/2209.02384v1)** | 2022-08-20 | [papers-cool](https://papers.cool/arxiv/2209.02384v1) |  |
| **[Beyond Visuals : Examining the Experiences of Geoscience Professionals With Vision Disabilities in Accessing Data Visualizations](http://arxiv.org/abs/2207.13220v1)** | 2022-07-27 | [papers-cool](https://papers.cool/arxiv/2207.13220v1) |  |

## Spatial
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[SITE: towards Spatial Intelligence Thorough Evaluation](http://arxiv.org/abs/2505.05456v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.05456v1) |  |
| **[Augmented Deep Contexts for Spatially Embedded Video Coding](http://arxiv.org/abs/2505.05309v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.05309v1) | 15 pages,CVPR |
| **[Thinning-Stable Point Processes as a Model for Spatial Burstiness](http://arxiv.org/abs/2505.00717v2)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.00717v2) | <details><summary>7 pag...</summary><p>7 pages, 2 figures. Accepted for WiOpt+25 Conference</p></details> |
| **[Spatially Disaggregated Energy Consumption and Emissions in End-use Sectors for Germany and Spain](http://arxiv.org/abs/2505.05139v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.05139v1) | <details><summary>11 pa...</summary><p>11 pages of text, 13 figures, 22 tables</p></details> |
| **[3DSRBench: A Comprehensive 3D Spatial Reasoning Benchmark](http://arxiv.org/abs/2412.07825v3)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2412.07825v3) | <details><summary>Proje...</summary><p>Project page: https://3dsrbench.github.io</p></details> |
| **[Anonymous Self-Stabilising Localisation via Spatial Population Protocols](http://arxiv.org/abs/2411.08434v2)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2411.08434v2) | <details><summary>Accep...</summary><p>Accepted as a brief announcement for SAND 2025</p></details> |
| **[SpatialPrompting: Keyframe-driven Zero-Shot Spatial Reasoning with Off-the-Shelf Multimodal Large Language Models](http://arxiv.org/abs/2505.04911v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.04911v1) | 18 pages, 11 figures |
| **[A Multi-Agent AI Framework for Immersive Audiobook Production through Spatial Audio and Neural Narration](http://arxiv.org/abs/2505.04885v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.04885v1) |  |
| **[Statistical Performance of Generalized Direction Detectors with Known Spatial Steering Vector](http://arxiv.org/abs/2505.03076v2)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.03076v2) | <details><summary>4 pag...</summary><p>4 pages,2 figures.This manuscript is accepted by Signal Processing Letters</p></details> |
| **[Texture Image Synthesis Using Spatial GAN Based on Vision Transformers](http://arxiv.org/abs/2502.01842v2)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2502.01842v2) | Preprint |
| **[Budgeted Spatial Data Acquisition: When Coverage and Connectivity Matter](http://arxiv.org/abs/2412.04853v3)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2412.04853v3) | <details><summary>The a...</summary><p>The article has been accepted by Frontiers of Computer Science (FCS), with the DOI: {10.1007/s11704-025-41427-9}</p></details> |
| **[PPO-ACT: Proximal Policy Optimization with Adversarial Curriculum Transfer for Spatial Public Goods Games](http://arxiv.org/abs/2505.04302v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04302v1) |  |
| **[Joint Task Offloading and Channel Allocation in Spatial-Temporal Dynamic for MEC Networks](http://arxiv.org/abs/2505.04272v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04272v1) |  |
| **[SUICA: Learning Super-high Dimensional Sparse Implicit Neural Representations for Spatial Transcriptomics](http://arxiv.org/abs/2412.01124v2)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2412.01124v2) | ICML 2025 |
| **[AS3D: 2D-Assisted Cross-Modal Understanding with Semantic-Spatial Scene Graphs for 3D Visual Grounding](http://arxiv.org/abs/2505.04058v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04058v1) |  |

## Time
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Crosslingual Reasoning through Test-Time Scaling](http://arxiv.org/abs/2505.05408v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.05408v1) |  |
| **[TiC-LM: A Web-Scale Benchmark for Time-Continual LLM Pretraining](http://arxiv.org/abs/2504.02107v2)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2504.02107v2) | <details><summary>Code ...</summary><p>Code available at: https://github.com/apple/ml-tic-lm</p></details> |
| **[Threshold Modulation for Online Test-Time Adaptation of Spiking Neural Networks](http://arxiv.org/abs/2505.05375v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.05375v1) | <details><summary>Accep...</summary><p>Accepted by IJCNN 2025. \c{opyright} 2025 IEEE. Personal use of this material is permitted. Permission from IEEE must be obtained for all other uses, including reprinting/republishing this material for advertising or promotional purposes, collecting new collected works for resale or redistribution to servers or lists, or reuse of any copyrighted component of this work in other works</p></details> |
| **[Time of the Flight of the Gaussians: Optimizing Depth Indirectly in Dynamic Radiance Fields](http://arxiv.org/abs/2505.05356v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.05356v1) |  |
| **[Progressive Inertial Poser: Progressive Real-Time Kinematic Chain Estimation for 3D Full-Body Pose from Three IMU Sensors](http://arxiv.org/abs/2505.05336v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.05336v1) |  |
| **[Generating Symbolic World Models via Test-time Scaling of Large Language Models](http://arxiv.org/abs/2502.04728v2)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2502.04728v2) | <details><summary>Accep...</summary><p>Accepted by TMLR2025 (32 pages, 6 figures)</p></details> |
| **[The Church Synthesis Problem over Continuous Time](http://arxiv.org/abs/2404.04782v3)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2404.04782v3) |  |
| **[Pairing Real-Time Piano Transcription with Symbol-level Tracking for Precise and Robust Score Following](http://arxiv.org/abs/2505.05078v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.05078v1) | <details><summary>5 pag...</summary><p>5 pages, 3 tables, 2 pseudocodes, to be published at the Sound and Music Computing Conference 2025</p></details> |
| **[T2S: High-resolution Time Series Generation with Text-to-Series Diffusion Models](http://arxiv.org/abs/2505.02417v2)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.02417v2) | <details><summary>Accep...</summary><p>Accepted by the 34th International Joint Conference on Artificial Intelligence (IJCAI 2025)</p></details> |
| **[Estimation of the long-run variance of nonlinear time series with an application to change point analysis](http://arxiv.org/abs/2404.02643v2)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2404.02643v2) | 32 pages, 2 figures |
| **[Do We Truly Need So Many Samples? Multi-LLM Repeated Sampling Efficiently Scales Test-Time Compute](http://arxiv.org/abs/2504.00762v4)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2504.00762v4) |  |
| **[Generative Models for Long Time Series: Approximately Equivariant Recurrent Network Structures for an Adjusted Training Scheme](http://arxiv.org/abs/2505.05020v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.05020v1) |  |
| **[Semi-Explicit Solution of Some Discrete-Time Mean-Field-Type Games with Higher-Order Costs](http://arxiv.org/abs/2505.04988v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.04988v1) | <details><summary>arXiv...</summary><p>arXiv admin note: text overlap with arXiv:2505.04112</p></details> |
| **[Robust Model-Based In-Hand Manipulation with Integrated Real-Time Motion-Contact Planning and Tracking](http://arxiv.org/abs/2505.04978v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.04978v1) | <details><summary>Submi...</summary><p>Submitted to the International Journal of Robotics Research (IJRR)</p></details> |
| **[A Survey of Slow Thinking-based Reasoning LLMs using Reinforced Learning and Inference-time Scaling Law](http://arxiv.org/abs/2505.02665v2)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.02665v2) |  |

## Trajectory
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[Efficient Estimation of Relaxed Model Parameters for Robust UAV Trajectory Optimization](http://arxiv.org/abs/2411.10941v3)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2411.10941v3) | <details><summary>8 pag...</summary><p>8 pages, 5 figures, to published in IEEE Sustech 2025</p></details> |
| **[An End-to-End Framework for Optimizing Foot Trajectory and Force in Dry Adhesion Legged Wall-Climbing Robots](http://arxiv.org/abs/2504.19448v2)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2504.19448v2) |  |
| **[FedTDP: A Privacy-Preserving and Unified Framework for Trajectory Data Preparation via Federated Learning](http://arxiv.org/abs/2505.05155v1)** | 2025-05-08 | [papers-cool](https://papers.cool/arxiv/2505.05155v1) |  |
| **[TrajEvo: Designing Trajectory Prediction Heuristics via LLM-driven Evolution](http://arxiv.org/abs/2505.04480v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04480v1) |  |
| **[Trajectory Entropy Reinforcement Learning for Predictable and Robust Control](http://arxiv.org/abs/2505.04193v1)** | 2025-05-07 | [papers-cool](https://papers.cool/arxiv/2505.04193v1) | 10 pages |
| **[DyTTP: Trajectory Prediction with Normalization-Free Transformers](http://arxiv.org/abs/2504.05356v2)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2504.05356v2) |  |
| **[Modeling Musical Genre Trajectories through Pathlet Learning](http://arxiv.org/abs/2505.03480v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03480v1) | <details><summary>Adjun...</summary><p>Adjunct Proceedings of the 33rd ACM Conference on User Modeling, Adaptation and Personalization (UMAP Adjunct '25)</p></details> |
| **[Reward-Based Collision-Free Algorithm for Trajectory Planning of Autonomous Robots](http://arxiv.org/abs/2502.06149v2)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2502.06149v2) |  |
| **[Generative Trajectory Stitching through Diffusion Composition](http://arxiv.org/abs/2503.05153v2)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2503.05153v2) | <details><summary>Proje...</summary><p>Project page: https://comp-diffuser.github.io/</p></details> |
| **[Inverse Dynamics Trajectory Optimization for Contact-Implicit Model Predictive Control](http://arxiv.org/abs/2309.01813v3)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2309.01813v3) | <details><summary>IJRR ...</summary><p>IJRR accepted version</p></details> |
| **[Trajectory Minimum Touching Ball](http://arxiv.org/abs/2505.02472v1)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2505.02472v1) |  |
| **[Riemannian Direct Trajectory Optimization of Rigid Bodies on Matrix Lie Groups](http://arxiv.org/abs/2505.02323v1)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2505.02323v1) | <details><summary>Accep...</summary><p>Accepted to Robotics: Science and Systems (RSS) 2025</p></details> |
| **[REPLAY: Modeling Time-Varying Temporal Regularities of Human Mobility for Location Prediction over Sparse Trajectories](http://arxiv.org/abs/2402.16310v4)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2402.16310v4) | <details><summary>Accep...</summary><p>Accepted by IEEE Transactions on Mobile Computing</p></details> |
| **[ARTEMIS: Autoregressive End-to-End Trajectory Planning with Mixture of Experts for Autonomous Driving](http://arxiv.org/abs/2504.19580v2)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2504.19580v2) |  |
| **[DiffOG: Differentiable Policy Trajectory Optimization with Generalizability](http://arxiv.org/abs/2504.13807v2)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2504.13807v2) |  |
